proc-macros/tests/ui/incorrect/sub/sub_no_item.rs

```diff
@@ -4,7 +4,7 @@ use jsonrpsee::proc_macros::rpc;
 #[rpc(client, server)]
 pub trait NoSubItem {
 	#[subscription(name = "sub")]
-	async fn sub(&self);
+	async fn sub(&self) -> RpcResult<()>;
```

Comment:
nit: Should we add a `sub_no_item.stderr` for this and the others missing the `.stderr`?

Reply:
we already got one and I think this happens before the `RpcResult` is checked but let's revert these UI tests

it should work for both `Result and ()` anyway